### ISSUE TITLE
subtitles selection is garbage

### ISSUE THREAD

**Issue by ikke-t** [hirsivaja/plugin.video.areena#39]
Hi,

I tried to watch Rauhantekijä from areena. I needed the subtitles and tried to select them. However the three available selections look more like some UUIDs. And none of them enable Finnnish subtitles. However, the same series downloaded by yle-dl show finnish subs just fine.

Kodi is of version 18.7.1 and this plugin is from Masters branch latest.

**Comment by ikke-t** [comment]
update, while I started to take logs, I noticed pihole is blocking cdnsecakmi.kaltura.com for some reason, which is used for fetching subtitles. So subtitles do work after all, but still the selection is gargabe. See selection from kore android client from which I took a screenshot. The selection looks the same in Kodi.

![image](https://user-images.githubusercontent.com/1146409/99155504-16eb6000-26c1-11eb-9ccd-168c690485a1.png)
